packages/core/src/array.ts

```diff
@@ -169,6 +169,24 @@ Array.prototype.reduce = function (callbackfn: any, initialValue: any) {
     return initialValue
 }
 
+Array.prototype.sort = function <T>(compareFn?: (a: T, b: T) => number) {
+    if (!compareFn) {
+        compareFn = (a: any, b: any) => a - b
```

Comment:
The default JS comparison function compares strings. For example:

```
> ["9", "80", "1"].sort()
[ '1', '80', '9' ]
> [9, 80, 1].sort()
[ 1, 80, 9 ]
> ["b", "z", "a"].sort()
[ 'a', 'b', 'z' ]
>
```

So I guess it should be sth like:

```
a=a + ""
b=b + ""
if (a < b) return -1;
else if (a > b) return 1;
else return 0;
```

Reply:
Thanks for the review. I made the changes including this one @mmoskal